handbook/communication/content_guidelines/style_and_mechanics.md

```diff
@@ -781,13 +781,13 @@ Use the natural plural/singular form. If a product or feature name is a plural n
 <div class="item yes">
 <h5>Yes</h5>
 <ul>
-<li>Batch Changes are available.</li>
+<li>Over 100 extensions are available.</li>
```

Comment:
I think Batch Changes is a better example for `Use the natural plural/singular form. If a product or feature name is a plural noun, treat it like a plural noun` The change made is showing the difference between capitalizing or not, which is not what's covered in this section.

Reply:
@content-hacker - I thought we treated Batch Changes as singular when referencing the product.

Batch Changes is now available.

Reply:
If that's the case then the section headline needs to be updated and we should treat all products this way (I don't think it's an issue with other ones). Is it confusing to say "we treat products as singular and features as plural?"

Reply:
Code Insights is now available would be the other. It does feel strange to treat these as singular. It doesn't look like we ever say Batch Changes is/are in any of the documentation/landing page/blog post.

Reply:
But you're right, "Code Insights" and "Batch Changes" are singular products. Isn't language fun 😆

Reply:
On https://about.sourcegraph.com/batch-changes we use Batch Changes in the singular "Batch Changes gives you a declarative structure for finding and modifying code across all of your repositories."

So the main things we want to convey are:
1. L1 products as defined in the product hierarchy use Title Case (when referencing the product)
2.  L1 products as defined in the product hierarchy are singular (when referencing the product).
3. Use natural plural/singular and lowercase when describing the functionality "The code insights I just generated are super useful."

Is that it?

CC @content-hacker @tommypez

Reply:
@abschu yes, this seems right to me.